opentracing-jdbc/opentracing-jdbc/src/test/java/org/zalando/opentracing/jdbc/Matrix.java

```diff
@@ -0,0 +1,50 @@
+package org.zalando.opentracing.jdbc;
+
+import org.h2.tools.SimpleResultSet;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Types;
+
+@SuppressWarnings("unused")
+// visible for testing
+public final class Matrix {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'Matrix' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/app/whiskeysierra/tracer/pullRequest?prid=4563925)